setup/mail-dovecot.sh

```diff
@@ -46,8 +46,45 @@ tools/editconf.py /etc/dovecot/conf.d/10-master.conf \
 # See http://www.dovecot.org/pipermail/dovecot/2013-March/088834.html.
 # A reboot is required for this to take effect (which we don't do as
 # as a part of setup). Test with `cat /proc/sys/fs/inotify/max_user_instances`.
+# Added IP Spoofing detection and other settings to harden network 
+# by preventing source routing of incoming packets and log malformed IP's 
 tools/editconf.py /etc/sysctl.conf \
 	fs.inotify.max_user_instances=1024
+	# IP Spoofing protection
+	net.ipv4.conf.all.rp_filter = 1
+	net.ipv4.conf.default.rp_filter = 1
+
+	# Ignore ICMP broadcast requests
+	net.ipv4.icmp_echo_ignore_broadcasts = 1
```

Comment:
I don't believe this is necessary, I don't see how harmful are ICMP broadcasts. In case of ICMP flooding abuse (or amplification attacks), the `fail2ban` configuration should protect the system.